### DISCUSSION TITLE
blog/spotify-refresh-token

### DISCUSSION THREAD

**Discussion by giscus** [pycoder2000/blog#29] [General]
# blog/spotify-refresh-token

Getting the Spotify Refresh Token doesn't need to be a hassle any longer.

https://musing.vercel.app/blog/spotify-refresh-token

**Comment by shouryan01** [comment]
test

**Comment by PrakasRavichandran** [comment]
Thank you Very Much. I tried this for long time. Finally I done it  💖.
